ironfish-cli/src/commands/service/bridge-relay.ts

```diff
@@ -0,0 +1,147 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { Assert, GetTransactionStreamResponse, Meter, TimeUtils, WebApi } from '@ironfish/sdk'
+import { Flags } from '@oclif/core'
+import { IronfishCommand } from '../../command'
+import { RemoteFlags } from '../../flags'
+
+export default class BridgeRelay extends IronfishCommand {
+  static hidden = true
+
+  static description = `
+    Relay Iron Fish deposits to the Sepolia bridge contract
+  `
+
+  static flags = {
+    ...RemoteFlags,
+    endpoint: Flags.string({
+      char: 'e',
+      description: 'API host to sync to',
+      parse: (input: string) => Promise.resolve(input.trim()),
+      env: 'IRONFISH_API_HOST',
+    }),
+    token: Flags.string({
+      char: 't',
+      description: 'API token to authenticate with',
+      parse: (input: string) => Promise.resolve(input.trim()),
+      env: 'IRONFISH_API_TOKEN',
+    }),
+    viewKey: Flags.string({
+      char: 'k',
+      description: 'View key to watch transactions with',
```

Comment:
We should label this as incoming view key, since we have the concept of a "view key" which has functionality of both ivk and ovk (see --export json)